lago/virt.py

```diff
@@ -36,6 +37,14 @@
 log_task = functools.partial(log_utils.log_task, logger=LOGGER)
 
 
+class LagoUnknownVMTypeError(utils.LagoUserException):
+    def __init__(self, vm_type_name, vm_types):
+        super().__init__(
+            'Unknown VM type: {0}, available types: {1}, \
+            need to install lago-ovirt plugin'.format(vm_type_name, vm_types)
```

Comment:
Didn't we agree to not mention `lago-ovirt` plugin, since it's possible that the `vm_type` will come from a different plugin?